### ISSUE TITLE
Does it work with CIELO BREEZ MAX?

### ISSUE THREAD

**Issue by HoLengZai** [bodyscape/cielo_home#19]
Hi

Anyone give a try with the CIELO BREEZ MAX?

I do not own any CIELO for the moment.
I have been using AmbiClimate for some years, but they recently got so many issues with their cloud, So I cannot control my AC for hours as I cannot login through their app.

I am planning to switch to Cielo even though it looks like that you must have a cielo account but at least they have physical buttons on the Cielo (compared with Sensibo which do not have)
You can use physical button without internet or cielo account right?

Thanks

**Comment by bodyscape** [comment]
Hi, you should ask your question on their website! They know all that.

**Comment by HoLengZai** [comment]
Indeed.. i have done that :)
What about your (wonderful) plugin... Anyone use it with the MAX version?

**Comment by bodyscape** [comment]
Honestly, i have no idea if it work with it.

**Comment by bodyscape** [comment]
If someone have it and let me connect to it. I can probably make it work.

**Comment by miles267** [comment]
Yes, I have been using with the Breez Max until just recently.  Devices now showing as unavailable though perhaps only temporarily.

**Comment by bodyscape** [comment]
@miles267 did you try the version 1.5.3?

**Comment by miles267** [comment]
> @miles267 did you try the version 1.5.3?

Yes, sir.  I've literally JUST updated to the latest 1.5.3 and it resolved my issue.  Thank you.  For some reason, I was looking in the default INTEGRATIONS page for a mention of an update.  It wasn't until I went into the HACS section that I was a Cielo integration upgrade available.
